client/src/cb_admin/pages/Signup.js

```diff
@@ -24,6 +24,7 @@ import {
   ORG_NAME_INVALID,
 } from '../constants/error_text';
 
+// TODO: get from api on mount
```

Comment:
Should we do this now, in case we forget later?

Reply:
Ya, can do